plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestKafkaWithConfluentSchemaRegistryMinimalFunctionality.java

```diff
@@ -145,11 +147,10 @@ public void testTopicWithRecordNameStrategy()
     }
 
     @Test
-    @Flaky(issue = "https://github.com/trinodb/trino/issues/6412", match = "Error registering Avro schema: .*")
     public void testTopicWithTopicRecordNameStrategy()
             throws Exception
     {
-        try (TestingKafka testingKafka = TestingKafka.createWithSchemaRegistry();
+        try (TestingKafka testingKafka = createWithSchemaRegistry();
```

Comment:
If the flaky issue is fixed then we no longer need to setup testing kafka and query runner per each test. Can you please remove it?